<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>setter</class>
 <widget class="QWidget" name="setter">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>324</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" rowspan="3">
    <widget class="pallete" name="widget" native="true"/>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>G = </string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>set like line color</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>set like back color</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QSlider" name="horizontalSlider">
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>R = </string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>B =</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QRadioButton" name="radioButton">
     <property name="text">
      <string>Grid</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>x^3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(x-4)^3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>(x+6)^3</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pallete</class>
   <extends>QWidget</extends>
   <header location="global">pallete.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
